velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -128,9 +135,21 @@ void checkReadErrorMessages(
   }
 }
 
-void verifyFailures(hdfsFS hdfs) {
-  HdfsReadFile readFile(hdfs, destinationPath);
-  HdfsReadFile readFile2(hdfs, destinationPath);
+bool checkMiniClusterStop(
+    ReadFile* readFile,
+    const std::string& errorMessage,
+    int endpoint) {
```

Comment:
nit: you don't need the endpoint. `readFile->pread(0, 1);` is sufficient.

Reply:
@majetideepak Yes. Removed the endpoint parameter.